src/KeyVault/KeyVault/help/Update-AzKeyVault.md

```diff
@@ -12,27 +12,45 @@ Update the state of an Azure key vault.
 
 ## SYNTAX
 
-### UpdateByNameParameterSet (Default)
+### UpdateKeyVaultByNameParameterSet (Default)
 ```
 Update-AzKeyVault -ResourceGroupName <String> -VaultName <String> [-EnableSoftDelete] [-EnablePurgeProtection]
  [-SoftDeleteRetentionInDays <Int32>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
  [<CommonParameters>]
 ```
 
-### UpdateByInputObjectParameterSet
+### UpdateManagedHsmByNameParameterSet
```

Comment:
This doc needs to regenerate, there should be no more HSM-related parameter sets